fpdcc/ccfp-asset-dashboard
# Add Phase model

## Overview

This PR adds a `Phase` model. A phase is a sub-project unit of work.

This PR intentionally omits material changes to the Project creation / editing interfaces. Will do in #80. It does update some view code so the related Phase and PhaseFinances objects are created when a project is added, but this implementation should not be considered final. I've left TODOs in places I've commented or otherwise made stopgap changes to the existing code.

Related to but doesn't quite close (?) #77.

## Testing Instructions

* This PR provides tests for the new data migration, both forwards and backwards. Review those tests to confirm I've implemented the data migration as we like, then confirm CI passes.
